test/statsCases/warnings-uglifyjs/conditional.js

```diff
@@ -0,0 +1,12 @@
+export function ifTrue() {}
```

Comment:
Please create a separate test case for this and don't put it into the `warnings-uglifyjs` case.